docs/non_containerized_NSFS.md

```diff
@@ -582,14 +582,14 @@ systemctl status rsyslog
 
 Noobaa logs are pushed to `var/log/noobaa.log` and the log is rotated and compressed daily.
 
-Verify the rsyslog and logrotate rpm configuration is complete by checking the files `etc/rsyslog.d/noobaa_syslog.conf` for rsyslog and `etc/logrotate.d/noobaa/logrotate_noobaa.conf` for logrotate.These files contain the noobaa specific configuration for rsyslog and logrotate.
+Verify the rsyslog and logrotate rpm configuration is complete by checking the files `etc/rsyslog.d/noobaa_syslog.conf` for rsyslog and `etc/logrotate.d/logrotate_noobaa.conf` for logrotate.These files contain the noobaa specific configuration for rsyslog and logrotate.
 
-Logrotate configuration is set up under `/etc/logrotate.d/noobaa/`. In order to trigger rotation when log files reache size threshold, each log file (`/var/log/noobaa.log` and `/var/log/noobaa_events.log`) uses its own rsyslog `outchannel` that triggers logrotate once file size reaches a limit (105MB). Logrotate is also typically called from cron on daily schedule.
+Logrotate configuration is set up under `/etc/logrotate.d/`.  Logrotate is called from cron on daily schedule.
 
 To rotate the logs manually run.
 
 ```
-logrotate /etc/logrotate.d/noobaa/logrotate_noobaa.conf
+logrotate /etc/logrotate.d/logrotate_noobaa.conf
```

Comment:
what about this doc -
https://github.com/noobaa/noobaa-core/blob/1e26ec2ec13cb9a98455e613673283f8207bef52/src/deploy/spectrum_archive/deployment_guide.md?plain=1#L150-L167